upgrade/Upgrade_Only_CSM_with_iuf.md

```diff
@@ -5,6 +5,29 @@ using IUF.
 
 ![Upgrade only CSM through IUF](../img/operations/diagram_upgrade_csm_with_IUF_101524.png)
 
+## Description
+
+The upgrade from CSM 1.5 to CSM 1.6 uses the IUF framework. The CSM upgrade steps are run automatically, either directly through IUF stages or by running a hook at the beginning or end of an IUF stage.
+The hooks that are run for the CSM upgrade are described below.
+
+- CSM upgrade prerequisites
+
+   CSM upgrade prerequisites are executed in a hook run before `pre-install-check`. This executes steps that are dependencies for later CSM upgrade steps.
+   This includes some service chart upgrades, uploading base NCN images to be used later in `prepare-images`, and other setup steps.
+   The specific script that is being executed is `/usr/share/doc/csm/upgrade/scripts/upgrade/prerequisites.sh`.
+
+- CSM services upgrade and validate CSM health
```

Comment:
I have the same comments on wording in this section as I did in the other section describing the CSM upgrade hooks executed within the IUF. It might be a maintenance headache to have the content duplicated across two files like this.